lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
@marcparadise it looks to me like the endpoint in question is not returning any JSON when it succeeds, is that intended?  https://github.com/chef/oc_chef_wm/blob/master/src/oc_chef_wm_org_associations.erl#L174 doesn't lead me to anything that looks like a response body anyway, it's been a while since I've messed with that code though :)

@poliva83 I think the answer here is we need to not parse the json in the response (since it's rather useless anyway).  I'll see if I can find the incantations to let .post() do that (post is what is trying to get json and failing).
 
Also, you don't have to add the { } if you don't want to, Ruby figures out what if you say a => b, you meant to have a hash around it.  Ruby is pliant like that, it's super nice until it isn't :)